docker/dev/tls/docker-compose.core-override.example.yml

```diff
@@ -57,8 +59,12 @@ services:
       # - ~/.step/certs:/usr/local/share/ca-certificates
 
   frontend:
+    environment:
+      __VITE_ADDITIONAL_SERVER_ALLOWED_HOSTS: "openproject-assets.local"
```

Comment:
Assuming that it's non-breaking for other users, could we specify this on the `docker-compose.yml` already? Then it would work out of the box for everyone using the default hostnames. If you want to use a different hostname, the override is still possible.